packages/schematics/angular/library/files/tsconfig.lib.json.template

```diff
@@ -9,6 +9,9 @@
     "inlineSources": true,
     "types": []
   },
+  "include": [
+    "**/*.ts"
```

Comment:
More explicate and allows users to add secondary entry-points without needing to update this tsconfig.

```suggestion
    "src/**/*.ts"
```